### PR TITLE
Update PEP 387 based on discussions

### PR DESCRIPTION
**Pull request by brettcannon**

### CODE REVIEW

File: pep-0387.txt [hunk 1]
```diff
@@ -48,31 +49,42 @@ This policy applies to all public APIs.  These include:
...
 
-- Function, class, module, attribute, method, and C-API names and types that
-  are prefixed by "_" (except special names).  The contents of these
-  are also not subject to the policy.
+- Function, class, module, attribute, method, and C-API names and
+  types that are prefixed by "_" (except special names).
+
+- Anything documented publicly as being private.
+
+- Imported modules (unless explicitly documented as part of the public
```
Comment (benjaminp) [inline pep-0387.txt:64]: Does this mean, for example, `import lzma; lzma.io.open()` isn't covered?

File: pep-0387.txt [hunk 2]
```diff
@@ -114,7 +126,8 @@ releases:
    reconsider.
 
 5. The behavior change or feature removal may now be made default or
-   permanent in the N+3 release.  Remove the old version and warning.
+   permanent having reached the declared default/removal version.
```
Comment (benjaminp) [inline pep-0387.txt:129]: I would just deleted "default/removal".

File: pep-0387.txt (hunk 1 above)
Comment (brettcannon) [reply pep-0387.txt:64]: Correct. Unless it's documented that `lzma.io` is a thing, it isn't backwards incompatible to remove it.

Comment (benjaminp) [reply pep-0387.txt:64]: Okay. It might be clearer to say "modules imported as a dependencies" or something.

**Pull request merged**